packages/components/datagrid/src/js/datagrid.controller.js

```diff
@@ -188,6 +191,10 @@ export default class DatagridController {
       this.hasTopbarContent = true;
     }
 
+    if (this.expandedRowElements && this.expandedRowElements.length) {
+      this.expandedRowCompiledTemplate = this.$compile(`<div>${this.expandedRowElements[0].innerHTML}</div>`);
```

Comment:
Why a wrapper `<div>` ? Didn't we have one a wrapper `<oui-datagrid-expanded-row>`.
If it's for style purpose, we could add a shim in the style for this tagname :)